content/en/docs/hertz/tutorials/observability/log.md

```diff
@@ -71,7 +71,68 @@ LevelWarn
 LevelError
 LevelFatal
 ```
+## hlog prints the log and specifies the field of the log
 
+Taking zerolog as an example, such a function is implemented in zerolog:
+
+```go
+package main
+
+import (
+    "bytes"
+    "github.com/cloudwego/hertz/pkg/common/json"
+    hertzZerolog "github.com/hertz-contrib/logger/zerolog"
+)
+
+func main() {
+    b := &bytes.Buffer{}
+    l := hertzZerolog.New(hertzZerolog.WithField("service", "logging"))
+    l.SetOutput(b)
+
+    l.Info("foobar")
+
+    type Log struct {
+        Level   string `json:"level"`
+        Service string `json:"service"`
+        Message string `json:"message"`
+    }
+
+    log := &Log{}
+
+    err := json.Unmarshal(b.Bytes(), log)//log.service=="logging"
+}
+```
+However, such functions are not directly implemented in zap and logrus, and the original option needs to be added manually
+
+Take zap as an example:
+```go
+package main
+
+import (
+	"bytes"
+	"github.com/cloudwego/hertz/pkg/common/json"
+	hertzzap "github.com/hertz-contrib/logger/zap"
+	"go.uber.org/zap"
+)
+
+func main() {
+	b := &bytes.Buffer{}
+	l := hertzzap.NewLogger(hertzzap.WithZapOptions(zap.Fields(zap.String("service", "logging"))))
+	l.SetOutput(b)
+
+	l.Info("foobar")
+
+	type Log struct {
+		Level   string `json:"level"`
+		Service string `json:"service"`
+		Message string `json:"message"`
+	}
+
+	log := &Log{}
+
+	err := json.Unmarshal(b.Bytes(), log) //log.service=="logging"
+}
+```
 ## Log Extension
 
-Currently, hlog supports the extended use of zap and logrus. For details on log extension, [see](https://www.cloudwego.io/docs/hertz/tutorials/framework-exten/log/).
+Currently, hlog supports the extended use of zap, logrus and zerolog. For details on log extension, [see](https://www.cloudwego.io/en/docs/hertz/tutorials/framework-exten/log/).
```

Comment:
There is no need to add /en here, which results in 404

Reply:
@ViolaPioggia PTAL and fix it.

Reply:
OK，I will fix it later